ansible_runner/runner.py

```diff
@@ -379,14 +379,16 @@ def host_events(self, host):
         return all_host_events
 
     @classmethod
-    def handle_termination(cls, pid, is_cancel=True):
+    def handle_termination(self, pid, pidfile=None, is_cancel=True):
```

Comment:
This is a class method, don't change this to `self`

Reply:
Got it, I hadn't been aware of the convention. Change in latest commit.